src/dappsExplorer/utils.ts

```diff
@@ -0,0 +1,17 @@
+import { DappV1, DappV2, isDappV2 } from 'src/dapps/types'
```

Comment:
i think i'm also in favour of not creating new "utils" files when possible, after hearing silas go on about this many times 😅 they tend to evolve into collections of miscellaneous and unfocused functions

Reply:
This will ultimately be shared by the search and filterSearch variants so I made a shared utils file. I am not sure of a better place for it if we want to share it between the two screens.